goalgorilla/open_social
# Issue #3254729 by yaroslav-shandra: Featured teasers does not support taxonomy custom icons

## Problem
In the resource library page the standard topics, events and groups icon is displayed. Not icon what was selected at topic/group/event type taxonomy term.

## Solution
Fix condition for topic/group/event type icons at templates. 

## Issue tracker
https://www.drupal.org/project/social/issues/3254729
https://getopensocial.atlassian.net/browse/SUP-1960

## How to test
1. Add term to Topic types taxonomy and select icon for term.
2. Create Topic with added term and include in the resource library.
3. Go to Resource library page and check added Topic icon.

## Screenshots
![Screenshot from 2021-12-16 14-52-06](https://user-images.githubusercontent.com/80881183/146379044-28186702-8e54-4cab-b49b-61b59e3569d1.png)
![Screenshot from 2021-12-16 14-53-34](https://user-images.githubusercontent.com/80881183/146379049-53d95b98-251d-4c80-97b8-084b0f737bbe.png)

## Release notes
Fixed Resource library template for showing icon selected from Topic type term.
